src/components/CommunityEvents/index.tsx

```diff
@@ -0,0 +1,229 @@
+//Libraries
+import React from "react"
+import { useI18next, useTranslation } from "gatsby-plugin-react-i18next"
+import {
+  Box,
+  Center,
+  Divider,
+  Flex,
+  Grid,
+  GridItem,
+  Heading,
+  Icon,
+  Text,
+} from "@chakra-ui/react"
+import { FaDiscord } from "react-icons/fa"
+import { DateTime, DateTimeFormatOptions } from "luxon"
+
+// Components
+import ButtonLink from "../ButtonLink"
+import Link from "../Link"
+import Translation from "../Translation"
+
+// Utils
+import { trackCustomEvent } from "../../utils/matomo"
+
+// Hooks
+import { useCommunityEvents } from "./useCommunityEvents"
+
+const matomoEvent = (buttonType: string) => {
+  trackCustomEvent({
+    eventCategory: "CommunityEventsWidget",
+    eventAction: "clicked",
+    eventName: buttonType,
+  })
+}
+
+const renderEventDateTime = (
+  date: string,
+  language: string,
+  params: DateTimeFormatOptions = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour12: false,
+    hour: "numeric",
+    minute: "numeric",
+  }
+) => {
+  return DateTime.fromISO(date).setLocale(language).toLocaleString(params)
+}
+
+const EventLink = (props: ComponentProps<typeof Link>) => (
+  <Link fontWeight="700" {...props} />
+)
+
+interface EventProps {
+  event: Event
```

Comment:
Noted that we are not using the `Event` type declaration that is declared in `useCommunityEvent`.

We should export the type first and then use it here:
```tsx
import { type Event as EventType } from "./useCommunityEvents"
```